tools/hack/create-cluster.sh

```diff
@@ -2,17 +2,24 @@
 
 set -euo pipefail
 
-## Create kind cluster.
-tools/bin/kind create cluster \
-    --name envoy-gateway 
+# Setup default values
+CLUSTER_NAME=${CLUSTER_NAME:-"envoy-gateway"}
+METALLB_VERSION=${METALLB_VERSION:-"v0.12.1"}
+NODE_IMAGE=${NODE_IMAGE:-"kindest/node:v1.24.0"}
```

Comment:
lets only make the `KIND_NODE_TAG` as the variable which corresponds to the kube version we'll be testing and fix the image to `kindest/node`

Reply:
updated